lib/jsonapi_compliable/scoping/filter.rb

```diff
@@ -52,8 +52,31 @@ def each_filter
         filter = find_filter!(param_name.to_sym)
         value  = param_value
         value  = value.split(',') if value.is_a?(String) && value.include?(',')
+        value  = normalize_string_values(value)
         yield filter, value
       end
     end
+
+    # Convert a string of "true" to true, etc
+    #
+    # NB - avoid Array(value) here since we might want to
+    # return a single element instead of array
+    def normalize_string_values(value)
+      if value.is_a?(Array)
```

Comment:
Type-checking like this always sends off my Spidey sense. So... two things:

1. Could you elaborate on the comment about _not_ using `Array(value)` here?
1. If we need to treat Array-like things differently than single items, maybe change this to:

    ```ruby
    if value.repond_to?(:to_ary)
      value.to_ary.map { |v| normalize_string_value(v) }
    else
      normalize_string_value(value)
    end
    ````